crates/pixi_manifest/src/pypi/pypi_options.rs

```diff
@@ -93,6 +123,20 @@ impl PypiOptions {
             self.index_url.clone()
         };
 
+        // Allow only one index strategy
+        let index_strategy = if let Some(other_index_strategy) = other.index_strategy.clone() {
+            if let Some(own_index_strategy) = self.index_strategy.clone() {
+                return Err(PypiOptionsMergeError::MultipleIndexStrategies {
+                    first: own_index_strategy.to_string(),
+                    second: other_index_strategy.to_string(),
+                });
```

Comment:
```suggestion
            if let Some(ref own_index_strategy) = self.index_strategy {
                return Err(PypiOptionsMergeError::MultipleIndexStrategies {
                    first: own_index_strategy.to_string(),
                    second: other_index_strategy.to_string(),
                });
```

Something like this?
I always forget the syntax.

Reply:
```suggestion
            if let Some(own_index_strategy) = &self.index_strategy {
                return Err(PypiOptionsMergeError::MultipleIndexStrategies {
                    first: own_index_strategy.to_string(),
                    second: other_index_strategy.to_string(),
                });
```